### PR TITLE
Add episode 50

### PR DESCRIPTION
**Pull request by colebow**
Keeping it real simple here.

I'll add a little bit of recap regarding what we talk about to the post once we've actually hosted the episode.

### CODE REVIEW

File: _episodes/50.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Concept of the episode: 50!
+
+No, that's not a factorial, we're just excited to have made it to 50 Trino
+Community Broadcast episodes. We've brought back some familiar faces to talk
+about what we've done, how we've got here, what it takes to keep an open source
+project ticking for over a decade, and celebrate the steps we've taken along
+the way. It's unscripted, and the discussion carries to wherever it feels like.
+
+Which means I'll fill in the rest of this recap once I know what we actually
```
Comment (bitsondatadev) [inline _episodes/50.md:54]: Maybe remove this now that you've filled it in already?

File: _episodes/50.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* Brian Olsen, Developer Advocate at [Tabular](https://tabular.io/)
```
Comment (bitsondatadev) [inline _episodes/50.md:24]: How did my title get changed back?

**Review by bitsondatadev — COMMENTED** [review]
Some small nits

**Review by mosabua — COMMENTED** [review]
Maybe add Starburst as sponsor mention and db-engines ranking and numbers?

**Review by mosabua — APPROVED** [review]

**Pull request merged**